packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -1,19 +1,41 @@
-import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
+import { makeResetStyles, makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { NavItemSlots, NavItemState } from './NavItem.types';
-import { typographyStyles } from '@fluentui/react-theme';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
 
 export const navItemClassNames: SlotClassNames<NavItemSlots> = {
   root: 'fui-NavItem',
   content: 'fui-NavItem__content',
+  icon: 'fui-NavItem__icon',
 };
 
+// These should match the constants defined in @fluentui/react-icons
+// This package avoids taking a dependency on the icons package for only the constants.
+const iconClassNames = {
+  filled: 'fui-Icon-filled',
+  regular: 'fui-Icon-regular',
+};
+
+const navItemTokens = { indicatorWidth: '4px', indicatorOffset: '-18px', indicatorHeight: '20px' };
```

Comment:
nit on formatting:
```suggestion
const navItemTokens = {
  indicatorWidth: '4px',
  indicatorOffset: '18px',
  indicatorHeight: '20px' 
};
```

I would also make the offset be a positive value. I usually treat vars like these as if they were tokens and therefore they should be implementation agnostic. This will also help if later you need that number to calculate something to avoid needing to do something like this: `calc(-1 * ${navItemTokens.indicatorOffset}`

Reply:
updated :-)